src/providers/push-notifications/push-notifications.ts

```diff
@@ -91,7 +91,10 @@ export class PushNotificationsProvider {
       );
       return;
     }
-    this._subscribe(walletClient);
+    if (!_.isArray(walletClient)) walletClient = [walletClient];
```

Comment:
We usually use `[].concat(walletClient)`

Reply:
+1!